cmd/clusterctl/configuration/ibmcloud/cluster.yaml.template

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
     clusterNetwork:
         services:
-            cidrBlocks: ["10.96.0.0/12"]
+            cidrBlocks: ["10.224.0.0/12"]
```

Comment:
Why update here? This is only for service IP, I think we can keep here unchanged?